trunk/SUAVE/Components/Wings/Wing.py

```diff
@@ -120,7 +120,7 @@ def __defaults__(self):
         self.transition_x_lower = 0.0
         
         self.Airfoil            = Data()
-        self.Segments           = SUAVE.Components.Wings.SegmentContainer()
+        self.Segments           = SUAVE.Core.ContainerOrdered()
```

Comment:
Kinda confused on what this does?

Reply:
It orders the segments, i.e. from segment 0 to N. But it also makes it iterable:

for segment in wing.Segments:
    some code...

Reply:
gotcha, im guessing this will be put into all segments?